tests/pipeline/test_domain.py

```diff
@@ -26,6 +33,20 @@
 from zipline.utils.pandas_utils import days_at_time
 
 
+DOMAINS = [
+    BE_EQUITIES,
```

Comment:
Probably the most important thing for us to test for each new domain is that its calendar code resolves as expected. The test for the EquityCalendarDomain defaults at https://github.com/quantopian/zipline/pull/2301/files#diff-8dd6ea1d2db90f77c7c793ecc4704888R339 do a good job of exercising that functionality, but right now that requires someone adding a new domain to remember to add it to the list in that test. One way to make it more likely that that happens would be to move this list to `domain.py` and rename it to something like `BUILT_IN_DOMAINS`. Then in this test, we could define the expected cutoff time for each domain and assert we have a definition for each builtin, so that if a user adds a domain and doesn't update the test, we'd fail.